articles/10-ways-to-contribute-to-geeksblabla.md

```diff
@@ -0,0 +1,122 @@
+---
+
+**"How can I contribute to the GeeksBlabla community?"** This is one of the most common questions we receive, and this blog post is here to answer it.
+
+Contributing to GeeksBlabla goes beyond being a core member—it’s about making an impact in ways that resonate with you, whether it’s by sharing feedback, suggesting topics, or being a guest on an episode. Some contributions, like giving a talk at BlablaConf, create a ripple effect that continues to inspire others long after.
+
+Whether you're a student exploring the tech world or a seasoned professional looking to give back, this guide will show you how you can make a difference while growing alongside the community. Let’s get started!
+
+## 1. Provide Feedback
+
+- **Impact:** ⭐⭐⭐
+- **Difficulty:** ⭐
+- **Time Investment:** ⭐
+
+Constructive feedback helps us improve. Whether it’s about our content, events, or initiatives, your input is invaluable in shaping the future of GeeksBlabla.
+
+Taking a few minutes to share your thoughts can lead to big improvements and better experiences for everyone involved.
+
+## 2. Spread the Word
+
+- **Impact:** ⭐⭐⭐⭐
+- **Difficulty:** ⭐
+- **Time Investment:** ⭐
+
+Help us grow the community by sharing GeeksBlabla content and initiatives with your network. By sharing, you help us discover talented individuals who might become future podcast guests or BlablaConf speakers, enriching our community with valuable knowledge and diverse perspectives. The larger our reach, the more opportunities we create for knowledge sharing and the greater our collective impact on Morocco's tech ecosystem.
+
+Tag us when you share, and invite others to join the conversation!
+
+## 3. Participate in Online Discussions
+
+- **Impact:** ⭐⭐⭐
+- **Difficulty:** ⭐
+- **Time Investment:** ⭐
+
+Whether it’s in the comments section of our live episodes or tweets by other community members on X or LinkedIn, sharing feedback on ideas and projects can create a ripple effect. Engaging in live episodes brings your perspective to light and enriches discussions.
+
+Similarly, your feedback amplifies the impact of others’ work. When you’re the one building, creating, or writing, the feedback you receive will push you forward. Cultivating the habit of providing and receiving feedback helps everyone grow together.
+
+## 4. Suggest Guests and Topics for Episodes
+
+- **Impact:** ⭐⭐⭐⭐
+- **Difficulty:** ⭐
+- **Time Investment:** ⭐
+
+One of the best ways to contribute to the GeeksBlabla podcast is by suggesting topics and guests. This helps with one of our most challenging tasks, since we rely on a broad network of connections to find great guests.
+
+You might know ideal guests or niche topics that the core team hasn't discovered yet, helping us fill blind spots we wouldn't otherwise see. Plus, you'll benefit from other members' suggestions, discovering new topics and voices.
+
+Check our [planning here](#) to suggest an episode or a guest for an episode.
+
+## 5. Be a Guest on an Episode
```

Comment:
I feel that 4 and 5 are kind of the same.

Reply:
can you check it now